lib/datasource/repology/index.ts

```diff
@@ -163,10 +164,10 @@ async function queryPackage(
 async function getCachedPackage(
   repoName: string,
   pkgName: string
-): Promise<RepologyPackage> {
+): Promise<RepologyPackage[]> {
   // Fetch previous result from cache if available
   const cacheKey = `${repoName}/${pkgName}`;
-  const cachedResult = await packageCache.get<RepologyPackage>(
+  const cachedResult = await packageCache.get<RepologyPackage[]>(
```

Comment:
Is it okay to change the data type in the cache? Or does it need to have some migration code to handle the old cache? I'm not sure about this so this is a question for the maintainers, but I think this might break if someone has an old cache. Or is the cache invalidated on every reonvate update?

Reply:
Yes, it would be best to change the `cacheKey` contents so that old formats are not returned. Either that or detecting if the returned value isn't an array and converting it.

Reply:
Do you have any preference? Changing cache key is of course simplest approach, but if you feel it's worth keeping it and handle non-array values I'm fine to try to do it.

Reply:
Ok, I decided to be on a safe side for version rollbacks and change cache key